build.gradle

```diff
@@ -64,8 +64,8 @@ sourceSets {
 dependencies {
     compile 'org.apache.commons:commons-lang3:3.5'
     compile 'org.apache.commons:commons-csv:1.4'
-    compile 'commons-net:commons-net:3.6'
     compile 'commons-io:commons-io:2.5'
+    compile 'com.github.seancfoley:ipaddress:5.3.1'
```

Comment:
Can't we use directly `java.net.Inet6Address` and avoid using an other lib?

Reply:
From project page of `com.github.seancfoley:ipaddress`:

```
Polymorphism is a key goal. The library maintains an address framework of interfaces and abstract classes that allow
most library functionality to be independent of address type of version, whether IPv4, IPv6 or MAC. This allows for code
which supports both IPv4 and IPv6 transparently. You can write generic non-version-specific code to validate
addresses, connect to addresses, produce address strings, mask addresses, etc. You can make use of the address
framework which is agnostic towards address version.
```

Writing a generic code that can work with both IPv4 and IPv6 made me prefer that library.

Reply:
But the actual reason of using that library is supporting both IP versions, unlike Apache Commons Net, which supports only IPv4

Reply:
Java standart library does not help with subnets, so we can't use `java.net.Inet6Address` directly, since IAB database includes a list of CIDR that needs to be processed